### PR TITLE
feat(session): support Oracle kill session

### PR DESCRIPTION
**Pull request by MarkPotato777**
#### What type of PR is this?

type-feature
#### What this PR does / why we need it:
This PR implements the Oracle kill session feature. Additionally, it refactors the `DefaultDBSessionManage` with the following changes: 
1. Removed the confusing logic of adding comments in SQL to indicate `ServerAddress`; 
2. Unified the kill logic within `DefaultDBSessionManage`, consolidating it into a single class.
#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-core/src/main/java/com/oceanbase/odc/core/sql/execute/SessionOperations.java [hunk 1]
```diff
@@ -30,4 +32,21 @@ public interface SessionOperations {
 
     void killQuery(@NonNull Connection connection, @NonNull String connectionId);
 
+    /**
+     * Get kill query SQL by connectionId
+     *
+     * @param connectionIds
+     * @return the map of connectionId to kill query SQL
+     */
+    Map<String, String> getKillQuerySqls(@NonNull List<String> connectionIds);
```
Comment (yhilmare) [inline server/odc-core/src/main/java/com/oceanbase/odc/core/sql/execute/SessionOperations.java:41]: the return value `Map<String, String>` is not easy to understand, how about this:
```Java
String getKillQuerySql(@NonNull String connectionId);
String getKillSessionSql(@NonNull String connectionId);
```

Comment (MarkPotato777) [reply server/odc-core/src/main/java/com/oceanbase/odc/core/sql/execute/SessionOperations.java:41]: Then the calling side needs to handle the scenario of getting SQLs in batch, which is not easy to use.
I think as the input parameter is a list, the return value, a map, does not seem that hard to understand.

Comment (yhilmare) [reply server/odc-core/src/main/java/com/oceanbase/odc/core/sql/execute/SessionOperations.java:41]: SessionOperations.java is used in SessionExtensionPoint which is our plugin interface and it is not a internal interface~, this interface may be impled by out developer. We should not design interfaces only from the perspective of our own use.

File: server/odc-core/src/main/java/com/oceanbase/odc/core/sql/execute/SessionOperations.java [hunk 2]
```diff
@@ -30,4 +32,21 @@ public interface SessionOperations {
...
+
+
+    /**
+     * Get kill session SQL by connectionId
+     *
+     * @param connectionIds
+     * @return the map of connectionId to kill session SQL
+     */
+    Map<String, String> getKillSessionSqls(@NonNull List<String> connectionIds);
```
Comment (LioRoger) [inline server/odc-core/src/main/java/com/oceanbase/odc/core/sql/execute/SessionOperations.java:50]: actually, input as list, but output as map, there maybe some question for me.
1. if output map key is connectionId, will input list guarantee connectionIds will not duplicate? if so , set is more suitable
2. if batch is implement by single query, provide single query method is needed.

Comment (MarkPotato777) [reply server/odc-core/src/main/java/com/oceanbase/odc/core/sql/execute/SessionOperations.java:50]: 👍🏻 Nice suggestions. I create two single methods:  `getKillQuerySql` and `getKillSessionSql`. The batch method will have a default implementation in the interface which relies on these two single methods. And the batch methods' input has been changed to a `Set`.

File: server/odc-core/src/main/java/com/oceanbase/odc/core/sql/execute/SessionOperations.java (hunk 1 above)
Comment (MarkPotato777) [reply server/odc-core/src/main/java/com/oceanbase/odc/core/sql/execute/SessionOperations.java:41]: I will change it by this way: 
I create two single methods: `getKillQuerySql` and `getKillSessionSql`. The batch method will have a default implementation in the interface which relies on these two single methods. And the batch methods' input has been changed to a `Set`.

**Review by yhilmare — APPROVED** [review]
lgtm

**Review by yhilmare — APPROVED** [review]
lgtm

**Review by yizhouxw — APPROVED** [review]
lgtm

**Pull request merged**